src/Tribe/Views/V2/Views/Widgets/Widget_View.php

```diff
@@ -221,7 +221,7 @@ public function get_html_classes( array $classes = [] ) {
 		 * @param array $html_classes Array of classes used for this widget.
 		 * @param View  $instance     The current View object.
 		 */
-		$html_classes = apply_filters( "tribe_events_views_v2_{$this->get_slug()}_widget_html_classes", $html_classes, $this );
+		$html_classes = apply_filters( 'tribe_events_views_v2_' . static::get_view_slug() . '_widget_html_classes', $html_classes, $this );
```

Comment:
Save into a variable, it's bad practice to build filter hooks directly on apply filters. E.g.:
```
$view_slug = ....;
$html_classes = apply_filters(  "tribe_events_views_v2_{$view_slug}_widget_html_classes" ...

```